shenyu-plugin/shenyu-plugin-general-context/src/main/java/org/apache/shenyu/plugin/general/context/GeneralContextPlugin.java

```diff
@@ -60,8 +60,9 @@
                         generalContextMapWithRpcType.put(each.getGeneralContextKey(), each.getGeneralContextValue());
                         break;
                     case Constants.TRANSMIT_HEADER_TO_GENERAL_CONTEXT_TYPE:
+                        final List<String> header = headers.get(each.getGeneralContextKey());
```

Comment:
Why this varable is final?

Reply:
> varable

If a parameter can be either mutable or immutable, it's better to make it immutable.